src/components/atoms/VideoOverlayButton/VideoOverlayButton.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import { useToggle } from "react-use";
+import {
+  faMicrophone,
+  faMicrophoneSlash,
+  faVideo,
+  faVideoSlash,
+  faVolumeMute,
+  faVolumeUp,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+
+import { ContainerClassName } from "types/utility";
+
+import { assertUnreachable } from "utils/error";
+
+import "./VideoOverlayButton.scss";
+
+export type VideoOverlayButtonVariant = "microphone" | "camera" | "audio";
```

Comment:
The syntax with union time here is just more concise. The difference I know of is that enums are present in runtime but unions are only compile-time TS constructs. Which enables you i.e. enumerate over all possible enum values. Which may be needed but not at this time at least.
But on the other hand you would have to write `VideoOverlayButtonVariant.xxx` instead of just strings which seems redundant to me. What do you think?

Reply:
for me, I have no problems writing more symbols like `VideoOverlayButtonVariant.xxx` as long as it gives more context to the type `VideoOverlayButtonVariant` which we use so that we could go there and check other possible values. Writing just "xxx", "yyy" doesn't give context to what we're iterating.

But I wouldn't say that I have a strong disagreement with Union Types. 
Here is the article according to which Union Types win. 
https://blog.bam.tech/developer-news/should-you-use-enums-or-union-types-in-typescript

But the main idea there, I'd say, is:
> But perhaps more importantly, keep in mind that this topic is a bit like tabs vs spaces: while it’s true that there are arguments for both, which one you’re using doesn’t matter that much in the end.

so I don't mind having union type here if you like. 🙃

Reply:
then keeping union here:) btw, thanks for the article!